content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,217 @@
+     - No
+   * - :doc:`Inventory valuation
+       <../../warehouses_storage/inventory_valuation/using_inventory_valuation>`
+     - Yes
+     - No
+     - No
+   * - Create transfer
+     - :ref:`Yes* <inventory/product_management/transfer-store>`
+     - :ref:`Yes* <inventory/product_management/transfer-con>`
+     - :ref:`No* <inventory/product_management/transfer-serv>`
+   * - :doc:`Lot/serial number tracking <differences>`
+     - Yes
+     - No
+     - No
+   * - Appears on :doc:`Replenishment report <../product_replenishment/strategies>`
+     - Yes
+     - :ref:`No* <inventory/product_management/replenish>`
+     - No
+   * - Can be manufactured or subcontracted
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - No
+   * - Can be in a kit
+     - Yes
+     - Yes
+     - No
+   * - Placed in package
+     - Yes
+     - :ref:`Yes* <inventory/product_management/package>`
+     - No
+   * - Appears on Inventory report
+     - Yes
+     - No
+     - No
+
+.. _inventory/product_management/on-hand-store:
+
+On-hand quantity
+----------------
+
+A storable product's on-hand and forecasted quantity based on incoming and outgoing orders are
+reflected on the product form, accessed by going to :menuselection:`Inventory app --> Products -->
+Products`.
+
+.. figure:: type/on-hand.png
+   :align: center
+   :alt: Show "On Hand" and "Forecast" smart buttons.
+
+   Current and forecasted quantities displayed in the **On Hand** and **Forecasted** smart buttons
+   on the product form.
+
+.. _inventory/product_management/on-hand-con:
+
+On the other hand, consumable products are regarded as always available, and they cannot be managed
+using reordering rules or lot/serial numbers.
+
+.. _inventory/product_management/transfer-store:
+
+Create transfer
+---------------
+
+*Transfers* are any warehouse operation, such as receipts, internal or batch transfers, or
+deliveries.
+
+When creating a transfer for storable products in the *Inventory* app, transfers modify the on-hand
+quantity at each location. For example, when five units of product is moved from the internal
+location `WH/Stock` to `WH/Packing Zone`, the five units recorded at `WH/Stock` is decreased, and
+five units recorded at `WH/Packing Zone` is increased.
+
+.. _inventory/product_management/transfer-con:
+
+For consumable products, although transfers can be created, the exact quantities at each storage
+location are not tracked.
+
+.. _inventory/product_management/transfer-serv:
+
+Service products cannot be included in transfers, but these products can be `linked to projects and
+tasks for deadline tracking <https://www.youtube.com/watch?v=fix2LGkv13c>`_.
+
+Replenishment
+-------------
+
+The replenishment report, accessed by going to :menuselection:`Inventory app --> Operations -->
+Replenishment` is a list of products that have a negative forecasted quantity and have manual
```

Comment:
```suggestion
Replenishment` is a list of products that have a negative forecasted quantity, **and** have manual
```
Just placing emphasis on this since both conditions are needed to appear here